rules/link.js

```diff
@@ -126,6 +127,11 @@ export default ({extensionURL = '', username = 'Unknown', clientID = '', clientS
 
   function promptUser() {
     return searchUsersWithSameEmail().then(function transformUsers(users) {
+      
+      users = users.filter(function(u) {
```

Comment:
Instead of reassigning the argument, can you just chain it into the command below? ex: `filter().map()`